_episodes/l1-05-continuous_integration.md

```diff
@@ -24,32 +24,468 @@ keypoints:
 
 ## Explanation of CI
 
-- event-driven computational workflows applied to your code
-- CI used as a gate-keeping device to enforce a common set of checks/quality on
-  commits
-- Many different CI systems e.g. CircleCI, TravisCI we're going to use GitHub
-  Actions because it's well integrated with GitHub.
-- Mention CD used to share e.g. publish.
+Continuous integration (CI) is a software development practice that ensures all
+contributions to a code base meet a defined standard. This is enforced via computational
+workflows that apply checks and tests to a code commit. Failure of these workflows to
+complete successfully is indicated via the code hosting platform and can be used to
+block code from being merged into branches.
```

Comment:
I'd mention an example or two in passing of a defined standard (e.g. whitespace formatting, unit tests etc.) to make it more concrete for students.

Reply:
Done